### PR TITLE
Add new FAQ covering versioning and maintenance approach

### PR DESCRIPTION
**Pull request by jcgraybill**
### Description
Add new FAQ covering versioning and maintenance approach. Also renaming two images to shorten file names to be NTFS compatible.
 
### Issues Resolved
https://github.com/opensearch-project/project-website/issues/661

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _faqs/03-28-which-version-of-opensearch-should-i-use.markdown [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+question: Which version of OpenSearch should I use?
+category: Upgrading to OpenSearch
+---
+
+or OpenSearch and other software in the OpenSearch project, new features and active development always takes place against the newest version. The OpenSearch project follows the [semantic versioning specification](https://opensearch.org/blog/technical-post/2021/08/what-is-semver/) for assigning version numbers to releases, so in most cases you should be able to upgrade to the latest version of any software without encountering incompatible changes.
```
Comment (elfisher) [inline _faqs/03-28-which-version-of-opensearch-should-i-use.markdown:6]: I think there is a missing "F" at the start of this paragraph

**Review by elfisher — CHANGES_REQUESTED** [review]
LGTM one minor suggestion.

File: _faqs/03-28-which-version-of-opensearch-should-i-use.markdown (hunk 1 above)
Comment (jcgraybill) [reply _faqs/03-28-which-version-of-opensearch-should-i-use.markdown]: 🤦‍♂️ Fixed, thanks.

**Review by elfisher — APPROVED** [review]
LGTM!

**Review by stockholmux — APPROVED** [review]
LGTM

**Pull request merged**